examples/set-default-labels-nested-package/README.md

```diff
@@ -0,0 +1,39 @@
+# set-default-labels: Nested Package Example
+
+### Overview
+
+This example shows how to run [`set-default-labels`] in a nested KPT package. 
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/set-default-labels-nested-package
+```
+
+Take a look at the `Kptfile`, it has the `set-default-labels` configured. Note, it does not require function configs (no `configPath` nor `configMap`).
+
+```yaml
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: your-app
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/set-default-labels:unstable
+```
+
+### Function invocation
+
+Invoke the function:
+
+```shell
+$ kpt fn render root-app
```

Comment:
> root-app

Should be `set-default-labels-nested-package`

Reply:
somehow deleted the root-app in the previous `kpt pkg get`. This command is correct.

Reply:
I somehow still can't see the `root-app` directory.

Reply:
I assume `root-app` is a subdirectory in `set-default-labels-nested-package`.
`cd set-default-labels-nested-package` is needed before running `kpt fn render root-app`.
Otherwise we need `kpt fn render set-default-labels-nested-package/root-app`

Reply:
missed a file during commit. Now it's added in `kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/set-default-labels-nested-package root-app`

Reply:
Got it! I didn't know it can be used this way.